app/models/setting/puppetdb.rb

```diff
@@ -20,14 +22,16 @@ def self.default_settings
     default_ssl_ca_file ||= (SETTINGS[:ssl_ca_file]).to_s
     default_ssl_certificate ||= (SETTINGS[:ssl_certificate]).to_s
     default_ssl_private_key ||= (SETTINGS[:ssl_priv_key]).to_s
+    default_api_version ||= 4
```

Comment:
I was thinking a major version release. If we want to avoid it, I can re-introduce this logic, but I think the settings module would be the suitable place for it.

What do you think?